src/other/pages/guides/getting-started/index.scss

```diff
@@ -0,0 +1,130 @@
+@import '../../utils/accessory-story.scss';
+@import '../../utils/canvas-content-padding.scss';
+@import '../../utils/last-text.scss';
+
+@include canvas-padding;
+@include content-padding;
+
+@include accessory-story-content;
+@include accessory-story-wrapper;
+
+.getting-started__subtitle {
+    @include text(t4, semibold);
+    text-transform: uppercase;
+    margin-bottom: -20px;
+    margin-top: 2.5rem;
+}
+
+.getting-started__last-text {
+    @include last-text();
+}
+
+.getting-started__header {
+    margin-top: 18px;
+}
+
+.getting-started__dependency-wrapper {
+    @include text(t2);
+}
+
+.getting-started__dependency {
+    background-color: #fef4f7;
+    box-shadow: 0 0 0 1px #f3a0bb;
+    color: #a71545;
+    border-radius: 2px;
+    font-size: 90%;
+    font-weight: 500;
+    padding: 2px 4px;
+}
+
+.getting-started__text {
+    @include text(t2);
+}
+
+.getting-started__text--important {
+    margin-bottom: 1.5rem;
+}
+
+
+.getting-started__text--bold {
+    @include text(t2, bold);
+}
+
+.callout__first-element {
+    margin-top: 2rem;
+}
+
+pre {
+    margin: 3rem 0;
+}
+
+// Duplicate to src/other/pages/utils/contact-footer/index.scss
```

Comment:
why duplicate?

Reply:
Already told the team in the daily. It was not possible for me to use the contact and callout Ad-hoc component in the jsx story.